### PR TITLE
Move RadioButton sample pages onto RadioButtons

### PR DESCRIPTION
**Pull request by jevansaks**
Change from StackPanel to RadioButtons to demonstrate the improved keyboarding and accessibility experience.

### CODE REVIEW

File: XamlControlsGallery/ControlPages/RadioButtonPage.xaml [hunk 1]
```diff
@@ -22,20 +23,18 @@
                     <ColumnDefinition />
                     <ColumnDefinition />
                 </Grid.ColumnDefinitions>
-                <StackPanel>
+                <muxc:RadioButtons Header="Options:">
                     <!-- A RadioButton group. -->
-                    <TextBlock Text="Options:" Style="{ThemeResource BaseTextBlockStyle}" />
                     <RadioButton x:Name="Option1RadioButton" Content="Option 1" Checked="Option1RadioButton_Checked" />
```
Comment (YuliKl) [inline XamlControlsGallery/ControlPages/RadioButtonPage.xaml]: Oh, so I can put something other than strings into the RadioButtons. @kikisaints, I definitely didn't get that from reading the spec, we'll need doc examples showing more than just x:String items.

**Comment by YuliKl** [comment]
@jevansaks, can you also mark the RadioButton page as "preview" (or "new") so its tile shows up correctly on the app's home page?

File: XamlControlsGallery/ControlPages/RadioButtonPage.xaml (hunk 1 above)
Comment (jevansaks) [reply XamlControlsGallery/ControlPages/RadioButtonPage.xaml]: You could, but what I'm doing here is moving the TextBlock that used to be the "header" and moving it to the Header property of RadioButtons. I didn't add a sample here (I'm hoping that @kikisaints will) showing how to databind to RadioButtons.

**Comment by jevansaks** [comment]
> @jevansaks Jevan Saks FTE, can you also mark the RadioButton page as "preview" (or "new") so its tile shows up correctly on the app's home page?

How do I do that? :)

File: XamlControlsGallery/ControlPages/RadioButtonPage.xaml (hunk 1 above)
Comment (YuliKl) [reply XamlControlsGallery/ControlPages/RadioButtonPage.xaml]: > > @jevansaks Jevan Saks FTE, can you also mark the RadioButton page as "preview" (or "new") so its tile shows up correctly on the app's home page?
> 
> How do I do that? :)

Add `"IsPreview": true,` to ControlInfoData.json near line 896

**Comment by YuliKl** [comment]
> > @jevansaks Jevan Saks FTE, can you also mark the RadioButton page as "preview" (or "new") so its tile shows up correctly on the app's home page?
> 
> How do I do that? :)

Add `"IsPreview": true,` to ControlInfoData.json near line 896

File: XamlControlsGallery/ControlPages/RadioButtonPage.xaml (hunk 1 above)
Comment (kikisaints) [reply XamlControlsGallery/ControlPages/RadioButtonPage.xaml]: @YuliKl This is a newer feature of RadioButtons as of their migration to the Repeater control. I'll add an example of that to the docs.

**Comment by gascardi** [comment]
@jevansaks It seems that your PR went in a long time ago, but I still don't see it in the latest version I can get from the Microsoft Store (1.2.11.0). This would have helped me resolve an accessibility issue.

image

Are releases in the Store not caught up with the releases of the master branch?
I'm using OS build 18363.535, Store app version 11912.1001.1.0.

Should I open an issue about this?
Thanks,
Gabriela

**Comment by stmoy** [comment]
Hi @gascardi! 

> Are releases in the Store not caught up with the releases of the master branch?

The version in the Store is out of date with what is currently in the master branch since we have a manual process to update the Store version. We generally do several Store updates a year (usually corresponding to big events like Build, Ignite, or updates to the WinUI package) but we haven't updated recently.

All of that said - I have been meaning to update the Store version, so thank you for this reminder 😅  I'll plan on doing this task early this week - and I'll provide an update here once I do so.

**Comment by gascardi** [comment]
Thanks, @stmoy! :-)
I usually check the app for the latest guidance; next time I'll check the source code as well.
Is there any version tagging when a build gets uploaded to the Store so I can check if the version I downloaded is up to date with the repo?

**Comment by stmoy** [comment]
Update: The version of the Gallery in the Store has been updated to the latest changes in master (as of Jan 13 2020). 

 > Is there any version tagging when a build gets uploaded to the Store so I can check if the version I downloaded is up to date with the repo?

That's a great idea! Since the process is manual right now, there isn't any automatic tagging, though this is something that I'd like to do at some point. In the meantime, building the source from the GitHub repo is the best way to get the latest-and-greatest.

**Comment by chingucoding** [comment]
> That's a great idea! Since the process is manual right now, there isn't any automatic tagging, though this is something that I'd like to do at some point. In the meantime, building the source from the GitHub repo is the best way to get the latest-and-greatest.

Something we could do is automatically publish to Microsoft Store after a commit to master using [this Azure Pipeline Task](https://marketplace.visualstudio.com/items?itemName=MS-RDX-MRO.windows-store-publish). What are your opinions on this @stmoy and @YuliKl ?

**Pull request merged**